moped-editor/src/views/projects/projectsListView/helpers.js

```diff
@@ -157,346 +134,277 @@ export const useColumns = ({ hiddenColumns }) => {
-        emptyValue: "-",
-        cellStyle: { whiteSpace: "noWrap" },
-        render: (entry) => {
-          return entry.contract_numbers.split(",").map((contractNumber, i) => (
-            <span key={i} style={{ display: "block" }}>
-              {contractNumber}
-            </span>
-          ));
-        },
+        renderCell: ({ row }) =>
+          renderSplitListDisplayBlock(row, "contract_numbers"),
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.medium,
       },
       {
-        title: "Tags",
+        headerName: "Tags",
         field: "project_tags",
-        hidden: hiddenColumns["project_tags"],
-        cellStyle: { whiteSpace: "noWrap" },
-        emptyValue: "-",
-        render: (entry) => {
-          return entry.project_tags.split(",").map((tag) => (
-            <span key={tag} style={{ display: "block" }}>
-              {tag}
-            </span>
-          ));
-        },
+        renderCell: ({ row }) =>
+          renderSplitListDisplayBlock(row, "project_tags"),
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.large,
       },
       {
-        title: "Created by",
+        headerName: "Created by",
         field: "added_by",
-        hidden: hiddenColumns["added_by"],
-        emptyValue: "-",
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.small,
       },
       {
-        title: "Public process status",
+        headerName: "Public process status",
         field: "public_process_status",
-        hidden: hiddenColumns["public_process_status"],
-        emptyValue: "-",
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.small,
       },
       {
-        title: "Interim MPD (Access) ID",
+        headerName: "Interim MPD (Access) ID",
         field: "interim_project_id",
-        hidden: hiddenColumns["interim_project_id"],
-        emptyValue: "-",
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.small,
       },
       {
-        title: "Components",
+        headerName: "Components",
         field: "components",
-        hidden: hiddenColumns["components"],
-        emptyValue: "-",
-        render: (entry) => filterComponentFullNames(entry),
+        renderCell: ({ row }) => filterComponentFullNames(row),
+        flex: 1,
+        minWidth: COLUMN_WIDTHS.medium,
       },
       {
-        title: "Parent project",
+        headerName: "Parent project",
         field: "parent_project_id",
-        hidden: hiddenColumns["parent_project_id"],
-        emptyValue: "-",
-        render: (entry) => (
+        renderCell: ({ row }) => (
           <Link
             component={RouterLink}
-            to={`/moped/projects/${entry.parent_project_id}`}
+            to={`/moped/projects/${row.parent_project_id}`}
             state={{ queryString }}
             sx={{ color: theme.palette.primary.main }}
           >
-            {entry.parent_project_name}
+            {row.parent_project_name}
           </Link>
         ),
+        flex: 1,
+        sortable: false,
```

Comment:
Part of me wonders if this should be pushed up into `COLUMN_CONFIG`? Idk I found finding the right mix of the existing `PROJECT_LIST_VIEW_QUERY_CONFIG` config and the config needed for the table library to be challenging when i was looking at this a few months ago.